python/datasources/cdc_hiv.py

```diff
@@ -138,9 +168,19 @@ def generate_breakdown_df(self, breakdown: str, geo_level: str, alls_df: pd.Data
                                            std_col.HIV_DIAGNOSES_PCT_SHARE,
                                            std_col.HIV_POPULATION_PCT,
                                            std_col.HIV_DIAGNOSES_PCT_INEQUITY)
+        df = generate_pct_rel_inequity_col(df,
```

Comment:
If you use the prefix/suffix pattern for col names you could put these into loops and simplify the code a little.